package.json

```diff
@@ -26,6 +26,7 @@
     "body-parser": "^1.18.2",
     "emotional": "0.0.3",
     "express": "^4.16.2",
+    "guardian-js": "^1.0.11",
```

Comment:
Why do you need an external node module to interact with the Guardian API?

Reply:
Easier to use...

Reply:
The [Guardian API](http://open-platform.theguardian.com/documentation/) looks pretty simple on its own 😃

Reply:
@claireinez while the guardian API is simple to use the client library should help as far as readability and reduction of tech debt goes. Otherwise you create your own functions that do the same thing... if I can make any changes to my client library to help you guys out let me know! I am going to work over this next week to reduce the size of the module as well as its dependencies (which should be none if I move to fetch + native promises). Thanks!

Reply:
@PorterK the library is straight forward, I would only suggests to add more examples in the readme. Thanks for your efforts and for responding to the issue.